modules/osd-create-cluster-ccs.adoc

```diff
@@ -204,30 +204,59 @@ The *Use a PrivateLink* option cannot be changed after a cluster is created.
 +
 .. If you are installing into an existing VPC and you want to enable an HTTP or HTTPS proxy for your cluster, select *Configure a cluster-wide proxy*.
 endif::osd-on-aws[]
+
 ifdef::osd-on-gcp[]
 . Optional: To install the cluster in an existing GCP Virtual Private Cloud (VPC):
 .. Select *Install into an existing VPC*.
 .. If you are installing into an existing VPC and you want to enable an HTTP or HTTPS proxy for your cluster, select *Configure a cluster-wide proxy*.
 endif::osd-on-gcp[]
-
++
 . Click *Next*.
 
+ifdef::osd-on-gcp[]
+. Optional: To install the cluster into a GCP shared VPC:
++
+[IMPORTANT]
+====
+
+To install a cluster into a shared VPC, you must use {product-title} version 4.13.15 or above. Additionally, the shared VPC administrator must enable a project as a host project in their Google Cloud console. For more information, see link:https://cloud.google.com/vpc/docs/provisioning-shared-vpc#set-up-shared-vpc[Enable a host project].
+====
+
+.. Select *Install into GCP shared VPC*.
+.. Specify the **Host project ID**. If the specified host project ID is incorrect, cluster creation will fail.
++
+
+[IMPORTANT]
+====
+Once you complete the steps within the cluster configuration wizard and click **Create Cluster**, the cluster will go into the "Installation Waiting" state. At this point, you must contact the VPC administrator of the host project, who must assign the dynamically-generated service account the following roles: **Computer Network Administrator**, **Compute Security Administrator**, and **DNS Administrator**.
+The administrator has 30 days to grant the listed permissions before the cluster creation fails.
+For information about shared GPC VPC permissions, see link:https://cloud.google.com/vpc/docs/provisioning-shared-vpc#migs-service-accounts[Provision shared VPC].
+====
+endif::osd-on-gcp[]
++
 . If you opted to install the cluster in an existing
 ifdef::osd-on-aws[]
 AWS
 endif::osd-on-aws[]
 ifdef::osd-on-gcp[]
 GCP
 endif::osd-on-gcp[]
-VPC, provide your *Virtual Private Cloud (VPC) subnet settings* and select *Next*. You must have created the Cloud network address translation (NAT) and a Cloud router. See the additional resources for information about Cloud NATs and Google VPCs.
+VPC, provide your *Virtual Private Cloud (VPC) subnet settings* and select *Next*.
```

Comment:
```suggestion
VPC, provide your *VPC subnet settings* and select *Next*.
```
Not sure why VPC was spelled out all the way down here. There are other, better "first mentions".

Reply:
![image](https://github.com/openshift/openshift-docs/assets/122639474/01f67332-984b-4476-bc3d-197a2df283d1)
Good point. FWIW, I think it was originally written this way b/c of the way it was presented in the UI.